access.go

```diff
@@ -162,36 +162,53 @@ func (s *Server) HandleAccessRequest(w *Response, r *http.Request) *AccessReques
 }
 
 func (s *Server) handleAuthorizationCodeRequest(w *Response, r *http.Request) *AccessRequest {
-	auth, err := CheckBasicAuth(r)
```

Comment:
Please help me to review @anggorodewanto @thoriqsatriya @banukusuma @rjjatson @desniaak